### DISCUSSION TITLE
iat test

### DISCUSSION THREAD

**Discussion by davedoesdev** [panva/jose#78] [Q&A]
Hi-
Nice project, I've switched over from jsjws.
I've got a question about https://github.com/panva/jose/blob/master/lib/jwt/verify.js#L270
What's the reasoning behind only checking `iat` is in the past if there's no `exp` claim?

**Comment by panva** [comment]
Hi @davedoesdev, see https://github.com/panva/jose/commit/0ed5025de30a754de95ae2587ce0f4573909b006
